coral/src/domain/notification/notification-api.ts

```diff
@@ -0,0 +1,12 @@
+import { KlawApiResponse } from "types/utils";
+import api from "src/services/api";
+import { transformGetNotificationCounts } from "src/domain/notification/notification-transformer";
+import { Notifications } from "src/domain/notification/notification-types";
+
+function getNotificationCounts(): Promise<Notifications> {
+  return api
+    .get<KlawApiResponse<"getAuth">>("/getAuth")
```

Comment:
Maybe this is not the right place, but I think a comment explaining why the `getAuth` endpoint is used here may be helpful. It confused me a moment and I had to check staging to be sure I understood the api definition and this call right 😬

Reply:
Added a comment in a separate commit.

Reply:
Awesome, thanks!